cfme/fixtures/smtp.py

```diff
@@ -54,7 +54,7 @@ def _finalize():
             return
         logger.info("Sending KeyboardInterrupt to collector")
         try:
-            collector.send_signal(signal.SIGINT)
+            subprocess.call(['pkill', '-INT', '-P', str(collector.pid)])
```

Comment:
we were killing shell which was calling python. basically we had 2 processes. e.g.
```
lkhomenk 19360  0.0  0.0   4496   764 ?        S    14:06   0:00 /bin/sh -c /home/lkhomenk/git/integration_tests/scripts/smtp_collector.py --smtp-port 1025 --query-port 1026
lkhomenk 19361  0.1  0.1 211216 22224 ?        Sl   14:06   0:00 python2 /home/lkhomenk/git/integration_tests/scripts/smtp_collector.py --smtp-port 1025 --query-port 1026
```
but were killing only sh.